### PR TITLE
Fixed Documentation Issues - 2020b 

### LINKED ISSUE #7926: Eng Ref documenation issue for ground heat transfer using C and F Factor methods
Issue overview
--------------
 - Equation 3.197 [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/engineering-reference/src/surface-heat-balance-manager-processes/ground-heat-transfer-calculations-using-c.tex#L73) for fictitious thermal resistance has incorrect sign. The double negative sign appears in equations 3.198 and 3.99 [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/engineering-reference/src/surface-heat-balance-manager-processes/ground-heat-transfer-calculations-using-c.tex#L97) and [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/engineering-reference/src/surface-heat-balance-manager-processes/ground-heat-transfer-calculations-using-c.tex#L113) as well. 

 - Underground wall effective heat transfer coefficient including the "floor construction" should have been "wall construction" [here](https://github.com/NREL/EnergyPlus/blob/develop/doc/engineering-reference/src/surface-heat-balance-manager-processes/ground-heat-transfer-calculations-using-c.tex#L126), This is most likely copied from section above but not edited.
 
## Details
Some additional details for this issue (if relevant):
 - All Platforms
 - Version of EnergyPlus V9.3

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### LINKED ISSUE #7949: Documentation Issues - 2020b
Issue overview
--------------
A collection of small documentation issues. See checkboxed comments below.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by jmythms**
Pull request overview
---------------------
 - Fixes #7949, #7926 
 - A small collection of fixes for documentation issues.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex [hunk 1]
```diff
@@ -212,35 +212,35 @@ \subsubsection{Field: DataElements}\label{field-dataelements}
...
+month & Month & - & Y \tabularnewline
+day & Day & - & Y \tabularnewline
+hour & hour & - & Y \tabularnewline
+minute & minute & - & N \tabularnewline
+datasource & datasource & - & N \tabularnewline
+drybulb & dry\_bulb\_temperature & C & Y \tabularnewline
+dewpoint & dew\_point\_temperature & C & Y \tabularnewline
+relhum & relative\_humidity & \% & Y \tabularnewline
+atmos\_pressure & atmospheric\_pressure & Pa & Y \tabularnewline
+exthorrad & extraterrestrial\_horizontal\linebreak\_radiation & $Wh/m^2$ & N \tabularnewline
```
Comment (mitchute) [inline doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex]: Lets typeset the units with the SI package (http://www.bakoma-tex.com/doc/latex/siunitx/siunitx.pdf). This one should be something like ```\si{\watt\hour\per\meter\squared}``` You might have to play around with it, but it should be something like that.

File: doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex [hunk 2]
```diff
@@ -362,7 +362,7 @@ \subsubsection{Date}\label{date}
 
 % table 11
 \begin{longtable}[c]{@{}lll@{}}
-\caption{Allowable hate formats for Custom Data entries. \label{table:allowable-hate-formats-for-custom-data}} \tabularnewline
```
Comment (mitchute) [inline doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex:365]: Nice.

**Review by mitchute — COMMENTED** [review]
One last question: does the ODG file you uploaded have all three figures? 

Let's make that minor change to use SI for the units, but other than that, this looks great!

File: doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex (hunk 1 above)
Comment (jmythms) [reply doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex]: I just changed it, could you check it please?

**Comment by jmythms** [comment]
> One last question: does the ODG file you uploaded have all three figures?
> 
> Let's make that minor change to use SI for the units, but other than that, this looks great!

Yes, the ODG file has all three figures.

File: doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex (hunk 1 above)
Comment (mjwitte) [reply doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex]: FYI, there are a few documentation style tips in the [wiki](https://github.com/NREL/EnergyPlus/wiki/BuildingEnergyPlus). Search for Documentation and go [here](https://github.com/NREL/EnergyPlus/wiki/Documentation). Feel free to add more.

Comment (jmythms) [reply doc/auxiliary-programs/src/weather-converter-program/definitions-file-custom-file-processing.tex]: @mjwitte Thanks! I have it bookmarked, will add to it if I come across something important.

**Comment by mitchute** [comment]
This looks great! And those shiny new actions-built LaTeX docs ready for review 5 minutes after pushing 🤩 . Merging.

**Comment by jmythms** [comment]
Thanks @mitchute !! :D

**Pull request merged**